docs/input-for-ledger-tool.md

```diff
@@ -0,0 +1,115 @@
+## Required Input Folder and Instruction JSON Files
+
+To debug your Solana program with Gimlet and Agave Ledger Tool, you **must** create an `input` folder in your project directory. For each instruction you want to test, add a `.json` file named after the instruction function. For example, for a Rust instruction function `pub fn initialize`, create:
+
+```
+input/initialize.json
+```
+
+If you have a multi program anchor project, you **must** create an program folder, should have the exact program name that this JSON file specify inputs for:
```

Comment:
If you have a multi-program anchor project, you **must** create a program folder, which must have the exact program name
that this JSON file specifies inputs for: